diff -ur docbook-to-man-2.0.0.orig/Instant/main.c docbook-to-man-2.0.0/Instant/main.c
--- docbook-to-man-2.0.0.orig/Instant/main.c	2025-02-21 00:14:34.947248962 +0100
+++ docbook-to-man-2.0.0/Instant/main.c	2025-02-21 00:20:01.078008760 +0100
@@ -88,6 +88,7 @@
 #include <sys/stat.h>
 #include <sys/file.h>
 #include <time.h>
+#include <unistd.h>
 
 #define STORAGE
 #include "general.h"
diff -ur docbook-to-man-2.0.0.orig/Instant/tptregexp/regerror.c docbook-to-man-2.0.0/Instant/tptregexp/regerror.c
--- docbook-to-man-2.0.0.orig/Instant/tptregexp/regerror.c	1999-07-03 14:58:24.000000000 +0200
+++ docbook-to-man-2.0.0/Instant/tptregexp/regerror.c	2025-02-21 00:24:40.810160974 +0100
@@ -1,4 +1,5 @@
 #include <stdio.h>
+#include <stdlib.h>
 
 void
 tpt_regerror(s)
diff -ur docbook-to-man-2.0.0.orig/Instant/tptregexp/regexp.c docbook-to-man-2.0.0/Instant/tptregexp/regexp.c
--- docbook-to-man-2.0.0.orig/Instant/tptregexp/regexp.c	2025-02-21 00:14:34.947720869 +0100
+++ docbook-to-man-2.0.0/Instant/tptregexp/regexp.c	2025-02-21 00:18:49.005720867 +0100
@@ -23,6 +23,7 @@
  * regular-expression syntax might require a total rethink.
  */
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 #include <tptregexp.h>
 #include "regmagic.h"
